Content.Server/Station/StationComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Robust.Shared.Analyzers;
+using Robust.Shared.GameObjects;
+using static Content.Server.Station.StationSystem;
+
+namespace Content.Server.Station;
+
+[RegisterComponent, Friend(typeof(StationSystem))]
+public class StationComponent : Component
+{
+    public override string Name => "StationJobList";
```

Comment:
name doesn't match the component class name?

Reply:
whoops, oldcode moment